parley/tests/README.md

```diff
@@ -13,4 +13,27 @@ If you think that everything is ok, you can start tests as follows:
 $ PARLEY_TEST="accept" cargo test
 ```
 
-It will update snapshots of the failed tests.
\ No newline at end of file
+It will update snapshots of the failed tests.
+
+## Usage of xtask when test fails
+
+After some tests fails, you may run the following for generating kompari HTML report:
+
+```bash
+cargo xtask report
```

Comment:
I believe a `.cargo/config.toml` is required with the following, as this currently doesn't work, but `cargo run --package xtask -- report` does.

```toml
[alias]
xtask = "run --package xtask --"
```

Reply:
You are completely right. I have just forgot to commit .cargo/config.toml file. Fixed now.